application_sdk/application/__init__.py

```diff
@@ -54,6 +55,11 @@ def __init__(
 
         self.client_class = client_class or BaseClient
         self.handler_class = handler_class or BaseHandler
+        
+        # MCP configuration
+        self.enable_mcp = enable_mcp
+        self.mcp_server = None
```

Comment:
let's just have one variable here, something like the following

`self.mcp_server = MCPServer() if enable_mcp else None`